### DISCUSSION TITLE
0.1 Release feedback

### DISCUSSION THREAD

**Discussion by bnema** [bnema/gordon#2] [General]
Let's discuss about this release

**Comment by prologic** [comment]
Hey @bnema 👋

I'm following your project because I'm interested in a lightweight solution for management app deployments. I don't have direct feedback based on usage just yet, because I'm unable to really use `gordon` as-is for the following reasons:

- No support for services (Swarm services)

I understand you're primarily using Podman, and to be honest I haven't really bothered looking into Podman, I have no compelling reason to really. I have several Docker Swarm clusters, thankfully with Traefik already part of its infrastructure. Without support for Swarm services, gordon wouldn't be suitable for use (yet). Is this something you think we could add support for?

This would also imply adding support for Config as well, as Docker Swarm also supports distributed configuration as well as Secrets.

**Comment by prologic** [comment]
Some other ideas I have in mind:

- I'd love it if we could have a `gordon.yaml` Docker Swarm stack that I could just `docker stack deploy` to my cluster to get Gordon going quickly/easily.
- It'd be great if we could also support OIDC authentication as I also already have an OIDC provider that I use internally (Authelia).

How do you also feel about abstracting the idea of "Apps" and a "Marketplace" of sorts? Docker/Microsoft tried to do this once with "App Bundles" but I believe the project failed. There are several attempts at this (CasaOS for example) but none I really quite like.
